src/collection.js

```diff
@@ -382,6 +382,15 @@ class Collection {
     return this.items.length
   }
 
+  /**
+   * Returns the max value in the collection.
+   *
+   * @returns {Number}
+   */
+  max() {

```

Comment:
Can you please a space between before the function parenthesis?

```suggestion
  max () {
```

Reply:
Done.